### PR TITLE
Issue 5 : Add openjdk-systemtest job to ci.adoptopenjdk jenkins

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Signed-off-by: Mesbah <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
add to whitelist

File: Build/jenkins/Jenkinsfile_openjdk-systemtest [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	environment {
+		PLATFORM='x64_linux'
+		MACHINE_LABEL='linux'
+        JCL_VERSION='current'
+        OPENJDK_TEST="$WORKSPACE/openjdk-test"
+        SPEC='linux_x86-64'
+        ANT_PATH="$WORKSPACE/systemtest_prereqs/apache-ant-1.10.1/bin"
+        OPENJDK_SYSTEMTEST_ROOT="$WORKSPACE/openjdk-test/openjdk-systemtest"
+        JAVA_HOME="$WORKSPACE/openjdkbinary/j2sdk-image/"
+        IS_SYSTEM_TEST=1
```
Comment (sophia-guo) [inline Build/jenkins/Jenkinsfile_openjdk-systemtest]: As this variable is only used once in setup stage I would suggest move it to setup stage. Can also just remove this variable and use "1" directly.

File: get.sh [hunk 2]
```diff
@@ -16,10 +16,14 @@ openjdktest=$(pwd)
...
 if [ "$#" -gt 2 ]
 then
-platform=$3
-jvm_version=$4
+	platform=$3
+	jvm_version=$4
 fi
 
+if [ "$#" -eq 5 ]; then
+	isSystemTest=$5
```
[the remark this answers is not in the document]
Comment (Mesbah-Alam) [reply get.sh]: Thanks. Updated.

**Comment by Mesbah-Alam** [comment]
Latest changes verified in this build: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/openjdk9_openj9_systemtest_x86-64_linux/76/console

**Pull request merged**